test/t_cose_compute_validate_mac_test.c

```diff
@@ -347,4 +347,228 @@ int_fast32_t compute_validate_get_size_mac_test()
     return 0;
 }
 
+
+/*
+ * Public function, see t_cose_compute_validate_mac_test.h
+ */
+int_fast32_t compute_validate_detached_content_mac_sig_fail_test()
+{
+    struct t_cose_mac_calculate_ctx   sign_ctx;
+    QCBOREncodeContext           cbor_encode;
+    int32_t                      return_value;
+    enum t_cose_err_t            result;
+    Q_USEFUL_BUF_MAKE_STACK_UB(  signed_cose_buffer, 300);
+    struct q_useful_buf_c        signed_cose;
+    struct t_cose_key            key;
+    struct q_useful_buf_c        payload;
+    QCBORError                   cbor_error;
+    struct t_cose_mac_validate_ctx verify_ctx;
+    size_t                       tamper_offset;
```

Comment:
tamper_offset is unused and should be removed.

Reply:
New commit [0b3db33](https://github.com/laurencelundblade/t_cose/pull/143/commits/0b3db3346b020a17adbb9098ca0dfb0d9678bae2) fixed this, and now `$ make -f Makefile.psa CMD_LINE=-Wall` shows no warning.